overlord/configstate/config/transaction.go

```diff
@@ -400,17 +400,18 @@ func commitChange(pristine *json.RawMessage, change interface{}) *json.RawMessag
 	panic(fmt.Errorf("internal error: unexpected configuration type %T", change))
 }
 
-// partOfVirtualConfiguration() return true if the requested key
-// is part of a virtual configuration. E.g.:
-// "true" for requestedKey: "a.virtual.b" when virtualKey is "a.virtual".
-func partOfVirtualConfiguration(virtualKey, requestedKey string) (bool, error) {
-	virtualSubkeys, err := ParseKey(virtualKey)
+// overlapsWith() return true if the requested key overlaps with the
+// given virtual key. E.g.
+// true: for requested key "a" and virtual key "a.virtual"
+// false for requested key "z" and virtual key "a.virtual"
+func overlapsWith(requestedKey, virtualKey string) (bool, error) {
```

Comment:
I was probably confusing but I suppose we need a longer name,  overlapsWithVirtual or overlapsWithVirtualConfig ?